applets/clock/clock.c

```diff
@@ -51,6 +51,13 @@
 #include <gdk/gdkkeysyms.h>
 #include <gio/gio.h>
 
+#ifdef CLOCK_INPROCESS
+#ifdef HAVE_WAYLAND
```

Comment:
could be `#if defined(CLOCK_INPROCESS) && defined(HAVE_WAYLAND)` to only have one block instead of nesting two

Reply:
selectors updated and they worked on a new build